sechub-developertools/scripts/sechub-api.sh

```diff
@@ -615,6 +659,8 @@ function sechub_user_signup_decline {
 
 # -----------------------------
 # main
+readonly JSON_TYPE='Content-Type: application/json'
```

Comment:
Here, a variable is introduced that is only used twice.

It is okay to define such a variable, but then every occurrence of `Content-Type: application/json` in the script should be replaced.
Please either remove the variable or replace the literal everywhere with the variable.

Reply:
Thought twice about it: 
I'd like it more to stay with the literal and remove the variable.
Then it's easier to compare with the `curl` examples on the documentation and one has not to look up the variable.